codegen/src/ClassCoordinates.test.scala

```diff
@@ -0,0 +1,59 @@
+package besom.codegen
+
+//noinspection ScalaFileName,TypeAnnotation
+class ClassCoordinatesTest extends munit.FunSuite {
+  implicit val providerConfig: Config.ProviderConfig = Config.ProviderConfig()
+
+  case class Data(
+    providerPackageParts: Seq[String],
+    modulePackageParts: Seq[String],
+    className: String,
+    tags: munit.Tag*
+  )(val expected: Expectations)
+  case class Expectations(
+    fullPackageName: String,
+    fullyQualifiedTypeRef: String,
+    filePath: String
+  )
+
+  val tests = List(
+    Data(
+      providerPackageParts = Seq("example"),
+      modulePackageParts = Seq(),
+      className = "Provider",
+      tags = munit.Ignore
```

Comment:
It looks like currently we check nothing because both test cases are ignored

Reply:
yes, all of the tests show unpatched bugs, I'd like to address bugs one by one